src/webgpu/api/operation/adapter/info.spec.ts

```diff
@@ -39,3 +39,42 @@ g.test('adapter_info')
       `adapterInfo.device should be a normalized identifier. But it's '${adapterInfo.device}'`
     );
   });
+
+g.test('same_object')
+  .desc(
+    `
+GPUAdapter.info provides the same object each time it's accessed`
+  )
+  .fn(async t => {
+    const gpu = getGPU(t.rec);
+    const adapter = await gpu.requestAdapter();
+    assert(adapter !== null);
+
+    const adapterInfo1 = adapter.info;
+    const adapterInfo2 = adapter.info;
+    t.expect(adapterInfo1 === adapterInfo2);
+  });
```

Comment:
Landed the spec, we should now have this test for device.adapterInfo too.

Reply:
Will do

Reply:
Done